.changeset/gentle-dots-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@optiaxiom/react": minor
+---
+
+Adding new Appshell component
```

Comment:
Squash this commit as changelog links to the commit where the changeset is added - rather than the PR